R/get_settings_profile.R

```diff
@@ -64,22 +62,30 @@ get_settings_profile <- function(parameters = c("NatM_uniform_Fem_GP_1", "SR_BH_
                                  high = c(0.40, 1.0, 2),
                                  step_size = c(0.01, 0.05, 0.25),
                                  param_space = c("multiplier", "real", "relative"),
-                                 use_prior_like = c(0, 0, 0)) {
+                                 use_prior_like = lifecycle::deprecated()) 
+{
+
   if (length(parameters) != length(low) |
     length(parameters) != length(high) |
     length(parameters) != length(step_size) |
-    length(parameters) != length(param_space) |
-    length(parameters) != length(use_prior_like)) {
+    length(parameters) != length(param_space)) {
     stop("Error: input vectors do match in length.")
   }
 
+  if (lifecycle::is_present(use_prior_like)) {
+    lifecycle::deprecate_warn(
+      #when = "1.46.0",
```

Comment:
Why is this line commented out? Also, this version number should be updated to the newly incremented version number.

Reply:
Thank you for catching this. I stole this chunk of code from r4ss and commented out this line since I had not looked at the version number for this package. I have added this line back in and will be incrementing the version to 1.1.2 for this pull request